org-code-javabuilder/playground/src/main/java/org/code/playground/TextItem.java

```diff
@@ -0,0 +1,140 @@
+ public class TextItem implements Item {
```

Comment:
clarifying question: text is never clickable?

Reply:
Correct.